index.md

```diff
@@ -191,48 +234,43 @@ Rbx<sub>n</sub> = 1.1 × BitRate
 
 If there is PES packet payload data in MB<sub>n</sub>, and buffer EB<sub>n</sub> is not full, the PES packet payload is transferred from MB<sub>n</sub> to EB<sub>n</sub> at a rate equal to Rbx<sub>n</sub>. If EB<sub>n</sub> is full, data are not removed from MB<sub>n</sub>. When a byte of data is transferred from MB<sub>n</sub> to EB<sub>n</sub>, all PES packet header bytes that are in MB<sub>n</sub> and precede that byte are instantaneously removed and discarded. When there is no PES packet payload data present in MB<sub>n</sub>, no data is removed from MB<sub>n</sub>. All data that enters MB<sub>n</sub> leaves it. All PES packet payload data bytes enter EB<sub>n</sub> instantaneously upon leaving MB<sub>n</sub>.
 
-### 4.5 STD delay
+### 4.7 STD delay
 
 The STD delay of any AV1 video through the system target decoders buffers TB<sub>n</sub>, MB<sub>n</sub>, and EB<sub>n</sub> shall be constrained by td<sub>n</sub>(j) – t(i) ≤ 10 seconds for all j, and all bytes i in access unit A<sub>n</sub>(j).
 
-### 4.6 Buffer management conditions
+### 4.8 Buffer management conditions
```

Comment:
I would imagine we would want to cover the removal of the startcodes as part of T-STD?

Reply:
T-STD is considering the AUs defined earlier in the spec, already embedding start code and emulation prevention. What is missing here is the impact of the "extra-bitrate" on the defined buffers. I'll add this in a revised version.

Reply:
Also something explaining that between MBn and EBn the startcodes are removed instantaneously I guess